standalone/src/main/java/io/jaegertracing/qe/CreateTraces.java

```diff
@@ -77,26 +76,25 @@
      */
     private static Tracer createJaegerTracer() {
         Tracer tracer;
-        Sender sender;
+        SenderConfiguration conf = null;
         CompositeReporter compositeReporter;
 
         if (USE_AGENT_OR_COLLECTOR.equalsIgnoreCase("agent")) {
-            sender = new UdpSender(JAEGER_AGENT_HOST, JAEGER_UDP_PORT, JAEGER_MAX_PACKET_SIZE);
+            conf = new SenderConfiguration().withAgentHost(JAEGER_AGENT_HOST).withAgentPort(JAEGER_UDP_PORT);
```

Comment:
It's probably better to configure it by just setting a system property. Like this:

https://github.com/jpkrohling/wildfly/blob/3a0cc555fd24dcbf176ab8d9267d35ba6543956f/microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/TracingDeploymentProcessor.java#L43-L50

Reply:
@jpkrohling With [this request](https://issues.jboss.org/browse/TRACING-173?focusedCommentId=13611258&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13611258), I may need to create multiple tracers with the different service name. I believe this approach will be good in that case.